src/test/kotlin/com/terraformation/backend/file/InMemoryFileStore.kt

```diff
@@ -10,11 +13,20 @@ import java.time.Instant
 import org.junit.jupiter.api.fail
 import org.springframework.http.MediaType
 
-class InMemoryFileStore(private val pathGenerator: PathGenerator? = null) : FileStore {
+@Named
+@Priority(1) // Higher priority than the non-test implementations of FileStore
+class InMemoryFileStore(private val pathGenerator: PathGenerator? = null) :
+    FileStore, BeanTestDouble {
   private var counter = 0
-  private val files = mutableMapOf<URI, ByteArray>()
   private val deletedFiles = mutableSetOf<URI>()
 
+  val files = mutableMapOf<URI, ByteArray>()
```

Comment:
Why can't this be private any more?

Reply:
It is used in a [test](https://github.com/terraware/terraware-server/pull/2030/files#diff-b218b08c6f9e7c40e78a31250114a35a42f6200602aebb2771044ff4fca4b9f1R126). I figured it would be OK since this is only used in test code.

[PDD source](https://github.com/terraware/pdd-backend/blob/main/server/src/test/kotlin/com/terraformation/pdd/file/InMemoryFileStore.kt#L23)